### PR TITLE
Update the bigquery connector doc

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
Update the bigquery connector doc

### CODE REVIEW

File: en/docs/reference/connectors/bigquery-connector/bigquery-connector-example.md [hunk 1]
```diff
@@ -506,26 +252,20 @@ You can download the ZIP file and extract the contents to get the project code.
```
Comment (chanikag) [inline en/docs/reference/connectors/bigquery-connector/bigquery-connector-example.md:252]: Is this file updated. Can you updated the zip file.

Comment (kalaiyarasiganeshalingam) [reply en/docs/reference/connectors/bigquery-connector/bigquery-connector-example.md:252]: Added now.

File: en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md [hunk 2]
```diff
@@ -1,66 +1,86 @@
-# Setting up the BigQuery Environment  
+# Set up the BigQuery environment  
 
 The BigQuery connector allows you to access the [BigQuery REST API](https://cloud.google.com/bigquery/docs/reference/rest) from an integration sequence.
 
 To work with the BigQuery connector, you need to have a Google Cloud Platform account. If you do not have a Google Cloud Platform account, go to [console.cloud.google.com](https://console.cloud.google.com/freetrial), and create a Google Cloud Platform trial account.
 
 BigQuery uses the OAuth 2.0 protocol for authorization. All requests to the BigQuery REST API will be authorized against a registered user. Developers can generate user credentials from the Google Cloud Platform using two different mechanisms. See the following sections for detail.
```
Comment (chanikag) [inline en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md:7]:
```suggestion
BigQuery uses the OAuth 2.0 protocol for authorization. All requests to the BigQuery REST API will be authorized against a registered user. Developers can generate user credentials from the Google Cloud Platform using two different mechanisms. See the following sections for details.
```

File: en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md [hunk 3]
```diff
@@ -1,66 +1,86 @@
...
+
+4. To create an `api key`, Click on **Create credentials** and then select **API key**.
+
+     <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-create-api-key.png" title="Select OAuth client ID" width="600" alt="Select OAuth client ID"/>
+
+5. Click on **Configure consent screen** in the next screen.
+
+     <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-consent-screen.png" title="Bigquery-credentials-page" width="300" alt="Bigquery-credentials-page"/>
+
+6. Provide the Application Name as `BigQueryExample`, and provide your email as `User support email` and `email address of Developer contact information`in the Consent Screen, then click on **SAVE AND CONTINUE**.
```
Comment (chanikag) [inline en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md:31]:
```suggestion
6. Provide the Application Name as `BigQueryExample`, and provide your email as `User support email` and `email address of Developer contact information` in the Consent Screen, then click on **SAVE AND CONTINUE**.
```

File: en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md [hunk 4]
```diff
@@ -1,66 +1,86 @@
...
+     <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-credentials-page.png" title="Bigquery-credentials-page" width="600" alt="Bigquery-credentials-page"/>   
    
-3. If you do not already have a project, you can create a new project. Click **Create credentials** and then select **OAuth client ID**.     
+3. If you do not already have a project, Enter `BigQueryExample` as the name of the project and click on **Create**.
+
+4. To create an `api key`, Click on **Create credentials** and then select **API key**.
+
+     <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-create-api-key.png" title="Select OAuth client ID" width="600" alt="Select OAuth client ID"/>
+
+5. Click on **Configure consent screen** in the next screen.
```
Comment (chanikag) [inline en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md:27]:
```suggestion
5. Click on the **Configure consent screen** in the next screen.
```

File: en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md [hunk 5]
```diff
@@ -72,59 +92,47 @@ Follow the steps below to generate user credentials.
...
-    <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-create-dataset1.png" title="Bigquery create Dataset step1" width="600" alt="Bigquery create Dataset step1"/> 
-
-2. Enter required Dataset details and click **Create Dataset**.
+    <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-select-project.png" title="Bigquery create Dataset step1" width="600" alt="Bigquery create Dataset step1"/> 
+   
+2. Click on the **View Action** on the created project.
+    
+    <img src="{{base_path}}/assets/img/integrate/connectors/bigquery-dataset.png" title="Bigquery create Dataset step2" width="600" alt="Bigquery create Dataset step2"/>
+   
+3. Enter required Dataset details and click **Create Dataset**.
```
Comment (chanikag) [inline en/docs/reference/connectors/bigquery-connector/bigquery-connector-configuration.md:129]:
```suggestion
3. Enter the required Dataset details and click **Create Dataset**.
```

File: en/docs/reference/connectors/bigquery-connector/bigquery-connector-example.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # BigQuery Connector Example
 
-The WSO2 BigQuery connector is mostly comprised of operations that are useful for retrieving BigQuery data such as project details, datasets, tables, and jobs (it has one operation that can be used to insert data into BigQuery tables).
+The WSO2 BigQuery connector is mostly composed of operations that are useful for retrieving BigQuery data such as project details, datasets, tables, and jobs (it has one operation that can be used to insert data into BigQuery tables).
 
 In this example we are trying to build up a sample scenario based on the BigQuery Table operations.
```
Comment (chanikag) [inline en/docs/reference/connectors/bigquery-connector/bigquery-connector-example.md:5]:
```suggestion
In this example, we are trying to build up a sample scenario based on the BigQuery Table operations.
```

**Review by chanikag — APPROVED** [review]

**Pull request merged**